mvcc/metrics.go

```diff
@@ -296,6 +296,14 @@ var (
 	// overridden by mvcc initialization
 	reportCompactRevMu sync.RWMutex
 	reportCompactRev   = func() float64 { return 0 }
+
+	putSizeGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: "etcd",
```

Comment:
Let's use `etcd_debugging` namespace to indicate it is an experimental (unstable) metric. This is typically how we add new metrics.

Reply:
Fixed. Thanks!